Add JDK12 prereqs to Ansible

Need some more header files on the build systems. These files were previously available in the OpenJDK source but from the latest JDK 12 code these files are not provided in their source, so they need to be available on the machine that is building OpenJDK.

Example failed configure output from a build on Cent6 x86-64
```
00:06:35 checking for X11/Intrinsic.h... yes
00:06:35 checking for X11/extensions/Xrandr.h... no
00:06:35 configure: error: Could not find all X11 headers (shape.h Xrender.h Xrander.h XTest.h Intrinsic.h). You might be able to fix this by running 'sudo yum install libXtst-devel libXt-devel libXrender-devel libXrandr-devel libXi-devel'.
00:06:35 configure exiting with result code 1
```

--- a/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/CentOS.yml
+++ b/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/CentOS.yml
@@ -30,6 +30,8 @@ Build_Tool_Packages:
   - libdwarf-devel
   - libpng-devel
   - libXext-devel
+  - libXi-devel # JDK12+ compilation
+  - libXrandr-devel # JDK12+ compilations
   - libXrender-devel
   - libXt-devel
   - libXtst-devel

--- a/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/FreeBSD.yml
+++ b/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/FreeBSD.yml
@@ -27,6 +27,7 @@ Build_Tool_Packages:
   - libXext
   - libXfixes
   - libXi
+  - libXrandr # JDK12+ compilation
   - libXrender
   - libXt
   - libXtst

--- a/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/OracleLinux.yml
+++ b/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/OracleLinux.yml
@@ -28,6 +28,8 @@ Build_Tool_Packages:
   - java-1.8.0-openjdk-devel
   - libdwarf
   - libXext-devel
+  - libXi-devel # JDK12+ compilation
+  - libXrandr-devel # JDK12+ compilation
   - libXrender-devel
   - libXt-devel
   - libXtst-devel

--- a/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/RedHat.yml
+++ b/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/RedHat.yml
@@ -30,6 +30,8 @@ Build_Tool_Packages:
   - libmpc-devel
   - libpng-devel
   - libXext-devel
+  - libXi-devel # JDK12+ compilation
+  - libXrandr-devel # JDK12+ compilation
   - libXrender-devel
   - libXt-devel
   - libXtst-devel

--- a/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/SLES.yml
+++ b/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/SLES.yml
@@ -29,6 +29,8 @@ Additional_Build_Tools_SLES12:
   - git
   - libfreetype6
   - libXext6
+  - libXi6 # JDK12+ compilation
+  - libXrandr2 # JDK12+ compilation
   - libXrender1
   - libXt6
   - libXtst6

--- a/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/Ubuntu.yml
+++ b/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/Ubuntu.yml
@@ -33,6 +33,8 @@ Build_Tool_Packages:
   - libwww-perl
   - libx11-dev
   - libxext-dev
+  - libXi-dev # JDK12+ compilation
+  - libXrandr-dev # JDK12+ compilation
   - libxrender-dev
   - libxt-dev
   - libxtst-dev

